account/manager.go

```diff
@@ -1568,6 +1678,22 @@ func (m *manager) spendAccount(ctx context.Context, account *Account,
 	return account, spendTx, nil
 }
 
+// deriveFeeFromPsbtPacket returns the transaction fee from a given PSBT packet.
+func deriveFeeFromPsbtPacket(packet psbt.Packet) btcutil.Amount {
```

Comment:
Same comment here about using a pointer directly. I get that it makes it more obvious that we're not modifying the packet here, but we don't really follow that pattern throughout the code base. Also, if you look at `psbt.Packet` it basically consists of pointer types only (a slice is a pointer), so this wouldn't protect the content of the package from being modified really anyway.

This could also just be called `deriveFeeFromPsbt()`. The `Packet` name is just something that's being used in the `psbt` package to make the naming nicer, otherwise it would be called `psbt.Psbt` which is discouraged by the Golang naming suggestions. So basically the `Packet` in this function name is kind of an implementation detail that can be omitted :)